nimbus.fml.yaml

```diff
@@ -0,0 +1,72 @@
+---
+channels:
+  - release
+  - beta
+  - nightly
+  - developer
+features:
+  homescreen:
+    description: The homescreen that the user goes to when they press home or new tab.
+    variables:
+      sections-enabled:
+        description: "This property provides a lookup table of whether or not the given section should be enabled.
+        If the section is enabled, it should be toggleable in the settings screen, and on by default."
+        type: Map<HomeScreenSection, Boolean>
+        default:
+          {
+            "top-sites": true,
+            "jump-back-in": false,
+            "recently-saved": false,
+            "recent-explorations": false,
+            "pocket": false
+          }
+    defaults:
+      - channel: nightly
+        value: {
+          "sections-enabled": {
+            "top-sites": true,
+            "jump-back-in": true,
+            "recently-saved": true,
+            "recent-explorations": true,
+            "pocket": true
+          }
+        }
```

Comment:
Is this going to affect the values returned from nimbus on [experiments.getVariables(FeatureId.HOME_PAGE, false)](https://github.com/mozilla-mobile/fenix/blob/6f9e725ec58f0658bb6cde057782991395bc3485/app/src/main/java/org/mozilla/fenix/experiments/NimbusFeatures.kt#L77)? At the moment we are [enabling everything by default](https://github.com/mozilla-mobile/fenix/blob/main/app/src/main/java/org/mozilla/fenix/experiments/NimbusFeatures.kt#L52-L57), except for pocket that it's only enabled in `"en-US"` and `"en-CA" ` .

Reply:
> At the moment we are [enabling everything by default](https://github.com/mozilla-mobile/fenix/blob/main/app/src/main/java/org/mozilla/fenix/experiments/NimbusFeatures.kt#L52-L57)

This is exactly how it should work: I'll update the file to reflect the current state.

Reply:
👍🏽 , just to confirm, did the last push update the default values?